styles/signupPage.css

```diff
@@ -0,0 +1,44 @@
+signup-container {
+  height: 80vh;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  max-width: 40rem;
+  margin: 1rem;
+  padding: 1rem 3rem;
+  margin: auto;
+  font-weight: 600;
+  gap: 1rem;
+  border-radius: 5px;
+  box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.3);
+}
+
+form input {
+  border: 1px solid var(--primary-color);
+  border-radius: 4px;
+  padding: 0.3rem 0.2rem;
+}
+
+form input::placeholder {
+  font-weight: 500;
+}
+
+form label {
+  margin-right: auto;
+  font-weight: 500;
+  font-size: 0.8rem;
+}
+
+form .signup-content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 0.4rem;
+}
+
+form a {
+  font-weight: 500;
+  font-size: 0.8rem;
+}
```

Comment:
Hey, do not apply styles directly on the elements, this is effect globally and if you make any form in e-comm, even if you do not want , it will apply these styles and you'll have to reset
Just add classes to everything.